File: Unit 3/IN300_WilliamAlecAkin_Unit3.java
/* William Alec Akin
Purdue University Global
IN300 - Programming for Data Analysis (Python, R, Java)
Dr. Robert Kayl
Unit 3 Assignment - Java Source Code
May 17th, 2022
*/

/* Description: Write a Java program, using ArrayList, and input the above data. Then, loop through the ArrayList and print
them all. Use the following data:

locomotive1 = steam
locomotive2 = diesel
locomotive3 = electric
locomotive4 = coal
locomotive5 = wood
common = combined engines
*/


//Importing the ArrayList class
import java.util.ArrayList;

//Creating the primary class (matches file name)
public class IN300_WilliamAlecAkin_Unit3 {
    
    //Creating the main method
    public static void main(String[] args) {
        
        //Create the array according to the assignment requirements
        ArrayList<String> trains = new ArrayList<String>();
        trains.add("locomotive1");
        trains.add("steam");
        trains.add("locomotive2");
        trains.add("diesel");
        trains.add("locomotive3");
        trains.add("electric");
        trains.add("locomotive4");
        trains.add("coal");
        trains.add("locomotive5");
        trains.add("wood");
        trains.add("common");
        trains.add("combined engines");

        //Loop through the ArrayList and print each element
        for (int i=0; i < trains.size(); i = i + 2) {
            System.out.println(trains.get(i) + "=" + trains.get(i+1));
        }

    }

}

File: Unit 3/IN300_WilliamAlecAkin_Unit3.py
'''William Alec Akin
Purdue University Global
IN300 - Programming for Data Analysis (Python, R, Java)
Dr. Robert Kayl
Unit 3 Assignment - Python Source Code
May 17, 2022'''

'''Description: Write a Python 3 program that uses sublists with the data below.
Then loop through the lists/sublists and print the values.

locomotive1=steam
locomotive2=diesel
locomotive3=electric
locamotive4=coal
locomotive5=wood
common=commined engines'''

# Create the lists of lists (sublists) with values above
trains = [["locomotive1", "steam"], ["locomotive2", "diesel"], ["locomotive3", "electric"], ["locomotive4", "coal"], 
        ["locomotive5", "wood"], ["common", "commined engines"]]

# Print the output to match the description
for i in range(len(trains)):
    print(trains[i][0] + "=" + trains[i][1])
